src/button/Button.story.mdx

```diff
@@ -0,0 +1,237 @@
+import { useState } from 'react'
+import { action } from '@storybook/addon-actions'
+import { Meta, Preview, Props } from '@storybook/addon-docs/blocks'
+
+import { color } from '../_utils/branding'
+import { Button, ButtonStatus } from '../button'
+import { ArrowIcon } from '../icon/arrowIcon'
+
+export const commonProps = {
+  onClick: action('clicked'),
+  onMouseDown: action('mouseDown'),
+  onMouseUp: action('mouseUp'),
+  onTouchStart: action('touchStart'),
+  onTouchEnd: action('touchEnd'),
+  onFocus: action('focused'),
+  onBlur: action('blured'),
+}
+
+export const renderPrimaryButton = () => {
+  const [buttonStatus, setButtonStatus] = useState(ButtonStatus.PRIMARY)
+  return (
+    <Button
+      status={buttonStatus}
+      {...commonProps}
+      onClick={() => {
+        setButtonStatus(ButtonStatus.LOADING)
+        setTimeout(() => setButtonStatus(ButtonStatus.CHECKED), 2000)
+        setTimeout(() => setButtonStatus(ButtonStatus.PRIMARY), 5000)
+      }}
+    >
+      Click to animate
+    </Button>
+  )
+}
+
+<Meta title="Buttons|Button" />
+
+# Button
+
+## Primary button
+
+> With text
```

Comment:
I know this format is widely used in our current doc but it is meant for quotes not for title. I would recommend using ### instead.

Reply:
All stories are impacted due to the generator we're using. Let's do a boyscout PR later for this